packages/calypso-e2e/src/jest-playwright-config/global-setup.ts

```diff
@@ -59,7 +59,7 @@ async function loginAndSaveCookies( testAccount: TestAccount, browser: Browser )
 	const page = await browser.newPage( pwConfig.contextOptions );
 	page.setDefaultTimeout( envVariables.TIMEOUT );
 	try {
-		await testAccount.logInViaLoginPage( page );
+		await testAccount.authenticate( page, { waitUntilStable: true } );
```

Comment:
**Key change**

All of the login-related stability updates have gone into `TestAccount.authenticate`. In fact, the previous method does only the username, password and TOTP (if any). 

From working on this for a couple of years, I know the authentication process for WordPress.com directs users through several intermediary pages before finally settling on the Reader/Home dashboard. The working hypothesis holds that authenticated cookie isn't actually "valid" until the user settles in to the Reader/Home dashboard endpoint[1].

What replacing `logInViaLoginPage` with `authenticate({waitUntilStable: true})` accomplishes is to apply all of stability upgrades meant to make authentication more reliable when warming cookies, so that subsequent E2E runs aren't using faulty cookies.

[1] This can be tested manually by opening F12 Developer Console, setting a throttle on the network for easier timing, initiating a login and then refreshing the browser before the authentication process completes. You will notice that you're booted out to the login screen again.